python/lsst/cp/pipe/__init__.py

```diff
@@ -35,3 +35,17 @@
 from .ptc import *
 from .utilsEfd import *
 from .version import *  # generated by sconsUtils unless you tell it not to
+
+import warnings
+from lsst.ip.isr.deferredCharge import OverscanModel as _OverscanModel
+from deprecated.sphinx import deprecated
+
+@deprecated(
+    reason="lsst.cp.pipe.OvercsanModel will be moved to lsst.ip.isr.deferredCharge.OverscanModel in v28.0.",
+    version="v28.0",
+    category=FutureWarning,
+)
+class OverscanModel(_OverscanModel):
+    pass
+
+del deprecated
```

Comment:
To be clear, we need to do this import/subclass/deprecate pattern for each of the moved classes.  We also need to file a ticket to clean up the deprecation, which should be a blocker for v29 release (if that exists as a ticket), and add a `TODO: Remove deprecated classes on DM-XXXXX`.  Also make sure there's a newline at the end of the file.